src/components/impact/styledImpact.js

```diff
@@ -0,0 +1,108 @@
+import styled from "styled-components";
+
+export const ImpactWrapper = styled.section`
+    background: #F4F4F4;
+    padding: 120px 0;
+    display: flex;
+    flex-direction: column;
+`
+export const BoxWrapper = styled.div`
+    display: flex;
+    justify-content: space-around;
+    margin: 60px auto 120px auto;
+`
+export const Box = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width: 45.8%;
+    align-items: center;
+    background: #FFFFFF;
+    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.2);
+    border-radius: 20px;
+`
+export const BoxTitle = styled.p`
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+    margin-top: 40px;
+`
+export const BoxText = styled.p`
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 21px;
+    text-align: center;
+    margin: 0 6.4%;
+
+    &:last-of-type {
+        margin-bottom: 60px;
+    }
+`
+export const BoxDiamond = styled.img`
+    width: 10px;
+    height: 10px;
+    margin: 20px 0;
+`
+// export const Grid = styled.div`
+//     width: 83.33%;
+//     display: grid;
+//     margin: 60px auto 0 auto;
+//     grid-template-columns: repeat(6, 1fr);
+//     grid-template-rows: repeat(8, 1fr);
+// `
+// export const GridCell = styled.div`
+//     border: 1px solid black;
+//     width: 200px;
+//     height: 80px;
+//     display: flex;
+//     align-items: center;
+//     justify-content: center;
+// `
+// export const GridText = styled.p`
+//     font-size: 18px;
+//     line-height: 21px;
+//     text-align: center;
+// `
+// export const GridPic = styled.img`
+// `
+export const Table = styled.table`
+    width: 83.33%;
```

Comment:
The 'single-use' plastics column is getting squeezed down to 188px while the other columns are 200px. I wonder if a min-width for the cells/columns can help here?